add_on_gateway.go

```diff
@@ -6,6 +6,7 @@ type AddOnGateway struct {
 	*Braintree
 }
 
+// All gets all addons from the gateway
```

Comment:
Could you add a full stop to the end of this comment and all other comments? I think you can drop the `from the gateway` also.

Reply:
@leighmcculloch no problem! Can i ask why a full stop might be necessary ?

Reply:
Because the text shows up as a paragraph on godoc, so it's good to have it be a complete sentence. E.g. https://godoc.org/github.com/lionelbarrow/braintree-go#AddressGateway.Update